store.go

```diff
@@ -3049,11 +3121,27 @@ func (s *store) Containers() ([]Container, error) {
 	return rcstore.Containers()
 }
 
-func (s *store) Layer(id string) (*Layer, error) {
-	lstore, err := s.LayerStore()
+func (s *store) Layer(id string) (layer *Layer, err error) {
```

Comment:
Looks like there's no need for named return parameters here.

Reply:
Fixed :+1: